setup.py

```diff
@@ -61,7 +61,7 @@
       install_requires=['numpy >= 1.12.0', 'katpoint >= 0.9', 'h5py >= 2.3', 'numba',
                         'katsdptelstate[rdb] >= 0.8', 'dask[array] >= 1.2.1',
                         'requests >= 2.18.0', 'pyjwt', 'future',
-                        'cityhash >= 0.2.2'],
+                        'cityhash >= 0.2.2', 'llvmlite==0.32.0'],
```

Comment:
A more appropriate change would be:
```
'llvmlite < 0.32; python_version < "3"'
```
llvmlite 0.32.0 is already too late, since it is Python 3 only. You want 0.31.0 (and any patch releases on top of that if it ever happens).

Reply:
awesome, thanks for background & the proposed refinement.